Formula/tmx.rb

```diff
@@ -0,0 +1,109 @@
+class Tmx < Formula
+  desc "Portable C library to load tiled maps in your games"
+  homepage "https://github.com/baylej/tmx"
+  url "https://github.com/baylej/tmx/archive/tmx_1.0.0.tar.gz"
+  sha256 "ba184b722a838a97f514fb7822c1243dbb7be8535b006ef1c5b9f928e295519b"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", "-DBUILD_SHARED_LIBS=on", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.tmx").write <<-EOS
```

Comment:
To keep the formula reasonable, can we use a `.tmx` file that is either loaded as a resource, or an example file from the tarball that we would install in `pkgshare`?

Reply:
I've cut down the size of the file used in the formula. If it's still not considered reasonable then let me know and I'll shift it elsewhere.